starter-code/css/main.css

```diff
@@ -7,4 +7,165 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+height: 100%;
+  width: 100%;
 }
+
+
+.css3gradient{width:1431px;height:1966px;
+ background-color:##06365f;
+ filter:progid:DXImageTransform.Microsoft.gradient(GradientType=0,startColorstr=#06365f, endColorstr=#ffffff);
+ background-image:-moz-linear-gradient(top, #06365f 19%, #ffffff 19%);
+ background-image:-webkit-linear-gradient(top, #06365f 19%, #ffffff 19%);
+ background-image:-ms-linear-gradient(top, #06365f 19%, #ffffff 19%);
+ background-image:linear-gradient(top, #06365f 19%, #ffffff 19%);
+ background-image:-o-linear-gradient(top, #06365f 19%, #ffffff 19%);
+ background-image:-webkit-gradient(linear, right top, right bottom, color-stop(19%,#06365f), color-stop(19%,#ffffff));}
+
+.divForPhones{
+
+height: 23%;
+width: 20%;
+float: left;
+margin: 160px;
+padding: 0px;
+
+
+}
+.phones{
+height: 100%;
+width: 100%;
+}
+.combined{
+  clear: both;
+  position: relative;
+
+  height: 100px;
+  width: 600px;
+  margin:  auto;
+  top: -600px;
+
+}
+.instagram{
```

Comment:
Make sure to properly space your code to make it easier to read. This is the standard format for CSS:

.button {
     width: 200px;
     height: 50px;
     top: 0px;
     margin: auto;
}

The indentation between the opening and closing brackets makes it easier for you, and especially for other people reading your code, to identify exactly what styles are being applied to the class name 'button'.